images/codesearch/cs-fetch-repos/src/fetch-repos.py

```diff
@@ -27,6 +27,9 @@
     'kubernetes-csi',
     'kubernetes-incubator',
     'kubernetes-sigs',
+    'kubernetes-nightly',
```

Comment:
please don't add the nightly and retired orgs!

Reply:
Understood, removed them now :)